### PR TITLE
feat: update theme

### PR DESCRIPTION
**Pull request by totoago**

### CODE REVIEW

File: packages/studio-website/src/pages/extension/left-side.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import CodeMirror from '@uiw/react-codemirror';
-import { useThemeContainer } from '@graphscope/studio-components';
+import { useCustomTheme } from '@graphscope/studio-components';
```
Comment (pomelo-nwu) [inline packages/studio-website/src/pages/extension/left-side.tsx:3]: 应该是 useCustomToken

File: packages/studio-website/src/pages/setting/International.tsx [hunk 2]
```diff
@@ -6,17 +6,19 @@ import type { ILocaleSwitchProps } from '@/components/locale-switch';
 import SettingParcel from '@/components/setting-parcel';
 
 const International: React.FunctionComponent = () => {
-  const { updateStore } = useContext();
-  const { handleTheme, locale = 'zh-CN' } = useThemeContainer();
-  const handleLocale = (value: ILocaleSwitchProps['value']) => {
+  const { store, updateStore } = useContext();
+  const { locale = 'zh-CN' } = store;
+  const { handleTheme } = useThemeContainer();
```
Comment (pomelo-nwu) [inline packages/studio-website/src/pages/setting/International.tsx:11]: useStudioProvider()
<StudioProvier></StudioProvier>

Comment (pomelo-nwu) [inline packages/studio-website/src/pages/setting/International.tsx:11]: const {changeTheme,changeLanguage} = useStudioProvier()  
 <StudioProvier />

File: packages/studio-website/src/pages/extension/left-side.tsx (hunk 1 above)
Comment (totoago) [reply packages/studio-website/src/pages/extension/left-side.tsx]: 已修改

File: packages/studio-website/src/pages/setting/International.tsx (hunk 2 above)
Comment (totoago) [reply packages/studio-website/src/pages/setting/International.tsx]: 已修改

**Pull request merged**